Documentation/image-fetching-behavior.md

```diff
@@ -0,0 +1,31 @@
+# Image fetching behavior
+
+When fetching, rkt will try to avoid unnecessary network transfers: if an updated image is already in the local store there's no need to download it again.
+
+This behavior can be controlled with the `--store-only` and `--no-store` flags.
+
+## General behavior
+
+The following table describes the meaning of the `--store-only` and `--no-store` flags.
+
+--store-only | --no-store |  Fetch from        | Description
```

Comment:
I think this table would be more readable if it just had two columns, the list of flags and the description. The default case should always come first.

--store-only | --no-store |  Fetch from        | Description
 ----------- | ---------- | ------------------ | -------------------------------------------------------------------------------
true         | false      | store              | Just check the store
false        | true       | remote             | Do remote download (handling caching logic for http(s):// and docker://)
false        | false      | store then remote  | Default Behavior. Do `store` and if it doesn't return an image do `remote`

| Flags | Description |
|---------|-----------------|
| _no flags_ | **Default behavior.** Do `store` and if it doesn't return an image do `remote`. |
| `--store-only` |  Check the local store only. | 
| `--no-store` | Execute a remote download, while handling caching logic for `http(s)://` and `docker://`. |
| `--store-only --no-store` | Invalid option. |